apps/www/src/view/searchForm.js

```diff
@@ -0,0 +1,33 @@
+import { fromPointer } from '@rdfine/hydra/lib/Collection'
+import { dash } from '@tpluscode/rdf-ns-builders'
+import { hyper_query as query } from '@hydrofoil/vocabularies/builders'
+import { html } from 'lit'
+
+export function searchForm(pointer) {
+  const collection = fromPointer(pointer)
+  const template = collection.search
+  const shape = template.get(dash.shape, { strict: false })
+  const currentFilters = collection.get(query.templateMappings, { strict: false })
```

Comment:
Quintessential usage of rdfine. `collection` is a proxy class which removes the need for `clownface` and importantly adds more than just property getters in the case of the implementation of `hydra:IriTemplate` `expand` method

Reply:
Interesting, I see the practical use of that.

Reply:
I will write a shape with a property named 'get' just to try it out :P

Reply:
It's a bit more complicated than that. SHACL is not supported now as I think you expect :)